stl/debugger/STL.natvis

```diff
@@ -1619,4 +1619,24 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     <Expand/>
   </Type>
 
+  <Type Name="std::span&lt;*&gt;">
+    <Intrinsic Name="data" Expression="_Mydata" />
+    <Intrinsic Name="size" Expression="_Mysize" />
+    <Intrinsic Name="size_bytes" Expression="_Mysize * sizeof(element_type)" />

```

Comment:
This is unused, and undiscoverable. Is there a good reason to have it here?

Reply:
It can be used in conditional breakpoints

Reply:
Maybe it would be a good idea to add `empty` intrinsic to every container visualizer. It might be quite useful for conditional breakpoint users.

Reply:
I suppose the discovery mechanism is reading the file. I don't imagine this will have a lot of users, but it's not hurting anything and no more likely to break than `size` so I won't complain.